providers/widget/test/main.js

```diff
@@ -0,0 +1,15 @@
+const utils = require('../../../utils');
+
+try{
+    utils.execStep(
+        'npx provider-cli test --configs-path=@marfeel/widgets-config',
```

Comment:
Since configs are common for widgets, i added all here instead of widget itself, wdyt @joselanderosmrf ?

Reply:
I think we should leave them at the widget level, this will give us more freedom to use different configs for some widgets if necessary and also if we don't have this in the package.json people won't be able to work in local.